resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/resources/session/Bug_266907.java

```diff
@@ -35,6 +36,10 @@ public static Test suite() {
 	}
 
 	public void test1stSession() {
+		// Ensure that no asynchronous save job restores the .project file after
+		// deleting it by suspending the JobManager for this workspace session
+		Job.getJobManager().suspend();
```

Comment:
It might be an overkill for this particular PR, but stopping the `JobManager` may even be desirable most (or even all) subclasses of `WorkspaceSessionTest`, right? 
I mean unless you're specifically testing the `JobManager` itself, it only gets in the way and runs unwanted logic between sessions (a.k.a. between test methods). Or am I seeing it wrong?

Reply:
I agree. That might even be of interest for all `ResourceTests` to improve test isolation and reduce the risk of unintended side effects.
W.r.t. to `WorkspaceSessionTest`: The benefit of these tests in comparison to `ResourceTests` is that a new `JobManager` is started for every session. So there is at least not the chance of leaking a job execution triggered in one test case into the next one.